mycore-iview2/src/main/java/org/mycore/iview2/iiif/MCRIVIEWIIIFImageImpl.java

```diff
@@ -76,38 +75,52 @@
     public static final String MAX_BYTES = "MCR.IIIFImage.Iview.MaxImageBytes";
 
     private static final String TILE_FILE_PROVIDER_PROPERTY = "TileFileProvider";
+    
+    private static final String THUMBNAIL_TILE_INFO_PROVIDER_PROPERTY = "ThumbnailTileInfoProvider";
 
     private static Logger LOGGER = LogManager.getLogger(MCRIVIEWIIIFImageImpl.class);
 
+    private static final String THUMBNAIL_PREFIX = "thumbnail:";
+    
     private java.util.List<String> transparentFormats;
 
     private MCRTileFileProvider tileFileProvider;
+    
+    private MCRThumbnailTileInfoProvider thumbnailTileInfoProvider;
 
     public MCRIVIEWIIIFImageImpl(String implName) {
         super(implName);
         Map<String, String> properties = getProperties();
 
         String tileFileProviderClassName = properties.get(TILE_FILE_PROVIDER_PROPERTY);
-
         if (tileFileProviderClassName == null) {
-            tileFileProviderClassName = MCRDefaultTileFileProvider.class.getName();
+            tileFileProvider = new MCRDefaultTileFileProvider();
+        } else {
+            Optional<MCRTileFileProvider> optTFP = MCRConfiguration2.getInstanceOf(tileFileProviderClassName);
```

Comment:
getInstanceOf() uses a property name and not a className as parameter. You might want to use instantiateClass() instead.